query/lib/src/bean/bean.dart

```diff
@@ -17,7 +17,7 @@ abstract class Bean<ModelType> {
   String get tableName;
 
   /// Creates a 'find' query
-  Find get finder => Sql.find(tableName);
+  Find get finder => Sql.find(Table(tableName));
```

Comment:
why ? non of the others take a `Table` so this one shouldn't either

Reply:
Someone changed the Find class. It is not possible to call it with with table name only (String)

Reply:
You're right... I'll ask because I think it's a mistake